financial_functions/lambda_handlers.py

```diff
@@ -334,3 +333,23 @@ def sln_handler(request, context):
     args = [request['cost'], request['salvage'], request['life']]
     return __call_ff('sln', args)
 
+def financial_function_generic_handler(request, context):
+    """
+    This function takes in an arbritary financial function and its parameters as inputs and returns the result of that calculation
+    :param request: Dict containing the financial function name and its parameters
+    :param context: Lambda execution context
+    :return: Dict with a 'result' entry containing the result of the calculation
+    """
+    logger.info("financial function request: {}".format(request))
+
+    function_name = request.get("functionName")
+    if function_name is None: 
+        return {"error": "Please provide a function name using the functionName parameter"}
+        
+    function_handler_name = function_name + "_handler"
+    if hasattr(sys.modules[__name__], function_handler_name):
```

Comment:
Be careful here. If someone passes `financial_function_generic` as the handler name you'll recursively call yourself to death. I was picturing putting this generic handler in a separate file/module and checking the module lambda_handlers for the corresponding handler function. I was thinking the other file would be called api_handler.py and would be expecting an API GW lambda proxy request, but I understand that's a TODO item for this. I would suggest moving to a diff file, but keeping the generic lambda so you're not wrestling with the API GW integration.